assets/css/less/wpbdp.less

```diff
@@ -1,138 +1,136 @@
-@import 'colors.less';
-@import 'common.less';
-@import 'wordpress-default-themes.less';
-@import 'datepicker.less';
+// Abstracts
+@variables-dir: "./wpbdp/abstracts/variables";
+@import "@{variables-dir}/box.less";
+@import "@{variables-dir}/typography.less";
+@import "./colors.less";
```

Comment:
Is there a reason to use "./colors.less"; instead of 'colors.less';? What is best practice here?

Reply:
You're correct! Using 'colors.less' is indeed the best practice. Thank you for your input!

Reply:
Thanks for checking! I'm not seeing any changes yet. Have your commits been pushed?

Reply:
I have been waiting for your input on changing the folder name.